rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -286,36 +299,24 @@ std::string lv2_fs_object::device_name_to_path(std::string_view device_name)
 					return fmt::format("%s%03d", mp->root, i);
 				}
 			}
+			break;
 		}
-		else
+		else if (mp->device == device_name || alias_check(mp))
 		{
-			if (mp->device == device_name || hdd_check(mp))
-			{
-				return std::string(mp->root);
-			}
+			return std::string(mp->root);
 		}
 	}
 
 	// Default fallback
 	return {};
 }
 
-bool lv2_fs_object::vfs_unmount(std::string_view vpath, bool no_error)
+bool lv2_fs_object::vfs_unmount(std::string_view vpath)
 {
 	const std::string local_path = vfs::get(vpath);
 
 	if (local_path.empty())
-	{
-		if (no_error)
-		{
-			return true;
-		}
-		else
-		{
-			sys_fs.error("\"%s\" is not mounted!", vpath);
```

Comment:
I don't see where this error message was moved to. Was it removed? If so why?
I keep noticing dev_hdd1 isn't mounted messages when I attempt to boot a game in xmb for testing purposes.
Therefore I think removing this message unless it is incorrect will fail to provide information that may be useful in developing the emulator in the future.

Reply:
I thought the built-in error report mechanism was enough for this purpose.
See line 3236~3237:
`
	if (!lv2_fs_object::vfs_unmount(vpath))
		return {CELL_ENOTMOUNTED, vpath};
`

Reply:
For instance: `SYS: 'sys_fs_unmount' failed with 0x8001003a : CELL_ENOTMOUNTED, /dev_hdd1 [1]`
The above message itself is self-explanatory enough.